.gitmodules

```diff
@@ -0,0 +1,32 @@
+[submodule "foldable-folder-list"]
+	path = foldable-folder-list
+	url = git@github.com:plancksecurity/android-foldable-folder-list.git
+	branch = main
```

Comment:
this is main and we merge it into develop, is it because we don't have a dev branch on andoird-foldable-folder-list?

Reply:
> this is main and we merge it into develop, is it because we don't have a dev branch on andoird-foldable-folder-list?

yes there is only main
we can create a develop branch though

Reply:
next time we touch it, not urgent